nixos/doc/manual/release-notes/rl-2211.section.md

```diff
@@ -73,6 +73,8 @@ Use `configure.packages` instead.
 
 - A new module was added for the Saleae Logic device family, providing the options `hardware.saleae-logic.enable` and `hardware.saleae-logic.package`.
 
+- The `framesh` new package has been added, which lets users interact with Web3 dapps.
```

Comment:
Another super tiny nit: "new package has been added" sounds weird. Maybe drop the "new"?